underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -395,8 +395,12 @@ public UfsFileStatus getFileStatus(String path) throws IOException {
     FileStatus fs = hdfs.getFileStatus(tPath);
     String contentHash =
         UnderFileSystemUtils.approximateContentHash(fs.getLen(), fs.getModificationTime());
+    long blockSize = fs.getBlockSize();
+    if (mUfsConf.getBoolean(PropertyKey.USER_BLOCK_SIZE_ENABLED)) {
```

Comment:
With this flag enabled, `getFileStatus()` will start returning wrong block size. So I don't think this the right place to patch up. 

What's the exact reason you want to override UFS block-size with Alluxio block-size?

Reply:
![image](https://user-images.githubusercontent.com/17329931/140917722-1db8b43c-6052-4d30-a152-efdc8dfa0a73.png)

I just reference the ObjectUnderFileSystem#getFileStatus method.

The exact reason for customize the block size is that in our alluxio cluster, cephfs is our underfilesystem, the blockSize from cephfs is 4MB, so a 1 GB big file will cost 1GB/4MB blocks, i want to customize the alluxio block size, for example 64MB, it would be better to reduce the block count.